pyproject.toml

```diff
@@ -8,9 +8,12 @@ description = "GIS is a Commandable SAL Component for the `Vera C. Rubin Observa
 license = { text = "GPL" }
 dependencies = ["pymodbus"]
 readme = "README.rst"
-urls = { documentation = "https://ts-gis.lsst.io", source_code = "https://github.com/lsst-ts/ts_gis"}
```

Comment:
Why this change? What is this based on? Our standards say to use `urls` here so please stick to those. If you think this can be improved then of course we'll be open to that but that needs to be discussed first with the rest of the team, for instance in parking lot or via Slack.